pkgs/development/libraries/qt-6/modules/qtsvg.nix

```diff
@@ -5,11 +5,20 @@
 , libmng
 , zlib
 , pkg-config
+, lib
+, stdenv
 }:
 
 qtModule {
   pname = "qtsvg";
   propagatedBuildInputs = [ qtbase ];
-  buildInputs = [ libwebp jasper libmng zlib ];
+  buildInputs = [
+    libwebp
+  ] ++ lib.optionals (!stdenv.hostPlatform.isMinGW) [
```

Comment:
```suggestion
  ] ++ lib.optionals (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
```

Reply:
This doesn't make sense to me. The buildPlatform can execute the hostPlatform? What does that mean?

Reply:
For example when you are cross compiling from x86_64-linux to i386-linux.

Reply:
But in most cases it just means cross compiling or not.

Reply:
Oh Okay that makes sense. I forget that stdenv here when I am trying to cross compile is pkgsCross.platform.stdenv